birdhouse/components/stac/default.env

```diff
@@ -3,6 +3,20 @@ export STAC_POSTGRES_PASSWORD=${POSTGRES_PAVICS_PASSWORD}
 export STAC_PGUSER=${POSTGRES_PAVICS_USERNAME}
 export STAC_PGPASSWORD=${POSTGRES_PAVICS_PASSWORD}
 
+# 'main' branch points at https://github.com/stac-utils/stac-fastapi/commit/d53e792
+# (see: https://github.com/crim-ca/stac-app/blob/40cad1aa7a094d58fca2d3184182761e248f781d/Dockerfile#L15-L20)
+# which corresponds to 2.4.3 release
+export STAC_VERSION=2.4.3-crim-main
+export STAC_IMAGE='ghcr.io/crim-ca/stac-app:main'
```

Comment:
Just curious, why not push the image with the actual tag `2.4.3-crim` instead of the generic tag `main`?

Reply:
See #371 and #346

Reply:
Yes, I could temporarily push a tag to make the reference more "semver"-like, but like @mishaschwartz highlighted, I would much rather have some work done on using the official image. The one tagged is already 9 months old. I don't want this patch tag to be considered a maintained revision.